lib/feed/widgets/feed_page_app_bar.dart

```diff
@@ -111,26 +111,30 @@ class FeedPageAppBar extends StatelessWidget {
         ),
         if (feedState.feedType == FeedType.community)
           PopupMenuButton(
-            onSelected: (value) async {
-              switch (value) {
-                case 'refresh':
-                  triggerRefresh(context);
-                  break;
-                case 'favorite':
-                  final Community community = context.read<FeedBloc>().state.fullCommunityView!.communityView.community;
-                  bool isFavorite = _getFavoriteStatus(context);
-                  await toggleFavoriteCommunity(context, community, isFavorite);
-                  break;
-              }
-            },
             itemBuilder: (context) => [
               PopupMenuItem(
-                value: 'refresh',
-                child: Text(l10n.refresh),
+                onTap: () => triggerRefresh(context),
+                child: Row(
```

Comment:
Would it be better to use `ListTile` in this case if we want to add in icons to the text?

It seems like that is the recommended approach when reading the documentation for [PopupMenuItem](https://api.flutter.dev/flutter/material/PopupMenuItem-class.html):

> Typically the [child](https://api.flutter.dev/flutter/material/PopupMenuItem/child.html) of a [PopupMenuItem](https://api.flutter.dev/flutter/material/PopupMenuItem-class.html) is a [Text](https://api.flutter.dev/flutter/widgets/Text-class.html) widget. More elaborate menus with icons can use a [ListTile](https://api.flutter.dev/flutter/material/ListTile-class.html).

Reply:
Sure thing, updated! I also modified some of the density properties of `ListTile` so that it takes up about the same amount of space as when using `Text`. (Otherwise, `ListTile` likes to be very big!)